server/src/main/java/com/flipkart/varadhi/services/SubscriptionService.java

```diff
@@ -14,13 +14,13 @@
 @Slf4j
 public class SubscriptionService {
     private final MetaStore metaStore;
-    private final ControllerApi controllerApi;
+    private final ControllerApi controllerClient;
```

Comment:
NIT: in general we can skip using the word client. client enforces the idea of it being remote, even though that is not a hard requirement.